frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx

```diff
@@ -158,7 +158,7 @@ export const ERDContentInner: FC<Props> = ({
         selectionOnDrag
         deleteKeyCode={null} // Turn off because it does not want to be deleted
       >
-        <CardinalityMarkers />
+        {!loading && <CardinalityMarkers />}
```

Comment:
[nits & imo]
Following the visibility control pattern used for TableNode loading, I think it would be better to apply the same approach to Edge components as well, as this would create more consistency in the implementation.

https://github.com/liam-hq/liam/blob/fix_cardinality_display_during_loading/frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.module.css#L11-L13

Reply:
That's good idea 👍 Thanks
b63e2dad86cd5f8509b44cf612c3bde1050e7c71